### PR TITLE
Expand coverage of test_addmm_sizes

### PR DESCRIPTION
**Pull request by zasdfgbnm**

### CODE REVIEW

File: test/test_torch.py [hunk 1]
```diff
@@ -16140,17 +16140,26 @@ def test_addmm_sizes(self, device, dtype):
...
-                    res2 += M
+                    res1 = torch.addmm(M, m1, m2, alpha=alpha, beta=beta)
+                    res2 = torch.full_like(res1, math.nan)
+                    torch.addmm(M, m1, m2, alpha=alpha, beta=beta, out=res2)
+                    res3 = (beta * M).cpu().tolist()
                     for i in range(n):
                         for j in range(m):
                             for l in range(k):
-                                res2[i, j] += m1[i, l] * m2[l, j]
+                                res3[i][j] += (alpha * m1[i, l] * m2[l, j]).item()
```
Comment (zasdfgbnm) [inline test/test_torch.py:16159]: The `item` is because of https://github.com/pytorch/pytorch/issues/43833

File: test/test_torch.py [hunk 2]
```diff
@@ -16140,17 +16140,26 @@ def test_addmm_sizes(self, device, dtype):
...
                     m1 = torch.randn(n, k, device=device, dtype=dtype)
                     m2 = torch.randn(k, m, device=device, dtype=dtype)
-                    res1 = torch.addmm(M, m1, m2)
-                    res2 = torch.zeros(n, m, device=device, dtype=dtype)
-                    res2 += M
+                    res1 = torch.addmm(M, m1, m2, alpha=alpha, beta=beta)
+                    res2 = torch.full_like(res1, math.nan)
+                    torch.addmm(M, m1, m2, alpha=alpha, beta=beta, out=res2)
+                    res3 = (beta * M).cpu().tolist()
                     for i in range(n):
```
Comment (ngimel) [inline test/test_torch.py:16156]: rather than doing this in a loop, maybe better compare with numpy matmul?

Comment (zasdfgbnm) [reply test/test_torch.py:16156]: If we compare with numpy, this means we will be unable to run this test without numpy

Comment (mruberry) [reply test/test_torch.py:16156]: That's OK. We will likely soon require NumPy to run the tests, anyway.

**Pull request closed without merging**